.pyre_configuration

```diff
@@ -10,10 +10,7 @@
     {"site-package": "flask_cors"},
     {"site-package": "freezegun"},
     {"site-package": "google/appengine"},
-    {"site-package": "google/cloud/datastore_v1"},
```

Comment:
We should also be able to remove things like `fakeredis` and `redis` here too, right? It also looks like I forgot to remove `rq`

Reply:
We still need redis momentarily (for raw memcache + flask handler caching).

Those will be easy enough to move out in separate PRs though